requirements-app.txt

```diff
@@ -0,0 +1 @@
+redis>=2.10.5
```

Comment:
Would prefer to move this to requirements.txt, as it is not only necessary for tests.